modules/migration-restic-errors.adoc

```diff
@@ -9,7 +9,7 @@
 [id='restic-timeout-error-velero-log_{context}']
 == Restic timeout error message in the Velero Pod log
 
-If a migration fails because Restic times out, the following error appears in the Velero Pod log.
+If a migration fails because Restic times out, the following error is displayed in the Velero Pod log.
```

Comment:
Can all of these changes of "appears" to "is displayed" stay in active voice? E.g., "the following error displays"?

Reply:
No. IBM is quite clear about that. "Displays" is not an intransitive verb.

Reply:
Ah ha, got it mixed up, I see it now. Thanks!